### PR TITLE
Hammer content credentials command

### PR DESCRIPTION
**Pull request by maximiliankolb**
Cherry-pick into:

* [x] Foreman 3.2
* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/modules/proc_importing-a-custom-gpg-key.adoc [hunk 1]
```diff
@@ -69,8 +69,9 @@ $ scp ~/etc/pki/rpm-gpg/RPM-GPG-KEY-_EXAMPLE-95_ root@{foreman-example-com}:~/.
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer gpg create \
---key ~/RPM-GPG-KEY-_EXAMPLE-95_ \
---name "_My_Repository_" \
---organization "_My_Organization_"
+# hammer content-credentials create \
+--content-type gpg_key \
```
Comment (ekohl) [inline guides/common/modules/proc_importing-a-custom-gpg-key.adoc:73]: Don't we usually indent the new line of a command to make it clear it's one command?

**Review by ekohl — COMMENTED** [review]
Comment not strictly related to this change.

File: guides/common/modules/proc_importing-a-custom-gpg-key.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_importing-a-custom-gpg-key.adoc:73]: We don't as far as I know.

**Review by melcorr — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Merged and cherry-picked: ✅ 
   469f60bf..be26b1f0  3.2 -> 3.2
   bc1c810f..81990f6e  3.1 -> 3.1

**Pull request merged**